### PR TITLE
Change around validator button order, choose default Docker image

### PR DESCRIPTION
**Pull request by nathanleclaire**
Two things that I think would help users on the validator page:

1. Change button order so "Run Validator" (the most important) is first
2. Select a Docker image by default (I've probably done this wrong on this go round, @SvenDowideit you'll have to tell me the right way to do it)

Signed-off-by: Nathan LeClaire <nathan@cryptoworkbench.io>

### CODE REVIEW

**Comment by nathanleclaire** [comment]
Screen Shot 2022-07-18 at 6 15 45 PM

File: src/renderer/nav/Validator.tsx [hunk 1]
```diff
@@ -38,7 +38,9 @@ const Validator = () => {
   const filterRef = useRef<HTMLInputElement>({} as HTMLInputElement);
   const validator = useAppSelector(selectValidatorNetworkState);
   const validatorImageName = 'cryptoworkbench/solana-amman';
-  const [validatorImageTag, setValidatorImageTag] = useState<string | null>('');
+  const [validatorImageTag, setValidatorImageTag] = useState<string | null>(
+    `${validatorImageName}:v1.10.31`
```
Comment (SvenDowideit) [inline src/renderer/nav/Validator.tsx:42]: horrible! and not worth changing for v0.4.0

if we're lucky, there's a simple API endpoint that tells solana validator's what the default mainnet version is, but i don't know of one atm -i don't want to parse github releases, discored etc :)

Comment (SvenDowideit) [reply src/renderer/nav/Validator.tsx:42]: see #235 :)

**Review by SvenDowideit — APPROVED** [review]
LGTM

File: src/renderer/nav/Validator.tsx (hunk 1 above)
Comment (nathanleclaire) [reply src/renderer/nav/Validator.tsx:42]: > horrible! and not worth changing for v0.4.0

😆 

> if we're lucky, there's a simple API endpoint that tells solana validator's what the default mainnet version is, but i don't know of one atm -i don't want to parse github releases, discored etc :)

Yea, picking the right one feels tricky. e.g., the most common one on mainnet right now is 1.10.29, but the most recent 1.10.* release is 1.10.31 🤷

**Pull request merged**